src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -563,4 +563,11 @@
   <data name="FailedToAnonymizeResource" xml:space="preserve">
     <value>Failed to anonymize resource. The job will be marked as failed. {0}</value>
   </data>
+  <data name="UnsupportedResourceType" xml:space="preserve">
```

Comment:
>UnsupportedResourceType [](http://example.com/codeflow?start=14&length=23)

there is `ResourceNotSupported` maybe we can reuse it?

Reply:
Good point! It is refined in the update.